pkg/server/plugin/nodeattestor/gcp/iit.go

```diff
@@ -17,74 +19,80 @@ import (
 const (
 	tokenAudience = "spire-gcp-node-attestor"
 	googleCertURL = "https://www.googleapis.com/oauth2/v1/certs"
+	svidPrefix    = "spiffe://{{ .TrustDomain }}/spire/agent"
 )
 
+var defaultAgentSVIDTemplate = template.Must(template.New("agent-svid").Parse(fmt.Sprintf("%s/{{ .PluginName}}/{{ .ProjectID }}/{{ .InstanceID }}", svidPrefix)))
+
 type tokenKeyRetriever interface {
 	retrieveKey(token *jwt.Token) (interface{}, error)
 }
 
+// IITAttestorPlugin implements node attestation for agents running in GCP.
+type IITAttestorPlugin struct {
+	svidTemplate      *template.Template
```

Comment:
nit: we should call this something else like idTemplate, idPathTemplate, or spiffeIDPathTemplate. SVID usually refers to the document itself, and one SVID type is X509 where golang already uses "template" term in the minting process so this name might be a bit confusing